analyzer/src/analysis/value/interprocedural/sequential/function_fixpoint.cpp

```diff
@@ -76,7 +77,17 @@ FunctionFixpoint::FunctionFixpoint(
       _checkers(checkers),
       _exit_invariant(make_bottom_abstract_value(ctx)),
       _return_stmt(nullptr),
-      _logger(logger) {}
+      _logger(logger)
+{
+  if (_ctx.opts.trace_ar_statements) {
+    ar::TextFormatter formatter{};
+    auto& stream = analyzer::log::msg().stream();
+    stream << "\n>>>>>>>>>>>>>>\nEntering Interprocedural Sequential FunctionFixpoint for ";
+    ar::Namer namer;
+    formatter.format_header(stream, _function, namer);
```

Comment:
I think you can construct the `ar::TextFormatter` in place.
```suggestion
    ar::TextFormatter().format_header(stream, _function, namer);
```